jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```
@@ -0,0 +1,30 @@
+pipeline {
+    agent none
+
+    parameters {
+
+        // https://datagrepper.engineering.redhat.com/id?id=ID:messaging-devops-broker02.web.prod.ext.phx2.redhat.com-44775-1580379079663-8:893046:0:0:1&is_raw=true&size=extra-large
+        string defaultValue: '', description: '', name: 'CI_MESSAGE', trim: true
```

```suggestion
        string defaultValue: '', description: 'This will be populated by the CI trigger', name: 'CI_MESSAGE', trim: true
```

^just a thought -- to match the parameter being set in the jjb yaml files